tests/python/common/test_heterograph_index.py

```diff
@@ -0,0 +1,97 @@
+import itertools
+import multiprocessing as mp
+import unittest
+from collections import Counter
+
+import backend as F
+
+import dgl
+import dgl.function as fn
+import networkx as nx
+import numpy as np
+import pytest
+import scipy.sparse as ssp
+import test_utils
+from dgl import DGLError
+from scipy.sparse import rand
+from test_utils import get_cases, parametrize_idtype
+from utils import assert_is_identical_hetero
+
+
+def create_test_heterograph(idtype):
+    # test heterograph from the docstring, plus a user -- wishes -- game relation
+    # 3 users, 2 games, 2 developers
+    # metagraph:
+    #    ('user', 'follows', 'user'),
+    #    ('user', 'plays', 'game'),
+    #    ('user', 'wishes', 'game'),
+    #    ('developer', 'develops', 'game')])
+
+    g = dgl.heterograph(
+        {
+            ("user", "follows", "user"): ([0, 1], [1, 2]),
+            ("user", "plays", "game"): ([0, 1, 2, 1], [0, 0, 1, 1]),
+            ("user", "wishes", "game"): ([0, 2], [1, 0]),
+            ("developer", "develops", "game"): ([0, 1], [0, 1]),
+        },
+        idtype=idtype,
+        device=F.ctx(),
+    )
+    assert g.idtype == idtype
+    assert g.device == F.ctx()
+    return g
+
+
+@unittest.skipIf(
+    F._default_context_str == "cpu", reason="Need gpu for this test"
+)
+@unittest.skipIf(
+    dgl.backend.backend_name != "pytorch",
+    reason="Pinning graph outplace only supported for PyTorch",
+)
+@parametrize_idtype
+def test_pin_memory(idtype):
+    # TODO: rewrite this test case to accept different graphs so we
```

Comment:
Do you have an issue to track the TODO? Usually I would suggest we follow TODO(#{ISSUE_ID}): blabla.

Reply:
Let me just remove this comment as it seems not related to this test...